ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -232,6 +219,103 @@ bool RSkComponentImage::processImageData(const char* path, char* response, int s
   return true;
 }
 
+inline void RSkComponentImage::drawContentShadow( SkCanvas *canvas,
```

Comment:
Add To DO for :When Frame doesn't have background, has border with Jpeg Image and no resize. currently drawing shadow for both border and content. Need to cross verify with reference and confirm the behaviour.